gengetopt.yaml

```diff
@@ -0,0 +1,142 @@
+package:
+  name: gengetopt
+  version: 2.23
+  epoch: 0
+  description: "A tool to write command line option parsing code for C programs"
+  copyright:
+    - license: GPL-3.0-or-later
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - build-base
+      - busybox
+      - libtool
+      - texinfo
+
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://ftp.gnu.org/gnu/gengetopt/gengetopt-${{package.version}}.tar.xz
+      expected-sha256: b941aec9011864978dd7fdeb052b1943535824169d2aa2b0e7eae9ab807584ac
+
+  - uses: autoconf/configure
+
+  - uses: autoconf/make
+
+  - uses: autoconf/make-install
+
+  - uses: strip
+
+subpackages:
+  - name: gengetopt-static
+    pipeline:
+      - uses: split/static
+    description: gengetopt static
+
+  - name: gengetopt-dev
```

Comment:
can we have `test/ldd-check` test here?

Reply:
What package should the ldd-check be for @Dentrax ?

Reply:
Sorry, i meant `pkgconf`: 

```
test:
      pipeline:
        - uses: test/pkgconf
```

if there are `so` files, we should also include `test/ldd-check` test.

Reply:
The pkgconf tests fail because there are no .po/pc files, so not sure

Reply:
@Dentrax pkgconf tests won't work as shown above but have added ldd-check